libraries/ui-library/package.json

```diff
@@ -39,29 +37,19 @@
     "@types/resize-observer-browser": "^0.1.5"
   },
   "devDependencies": {
-    "@fontsource/noto-sans": "^4.5.11",
+    "@fontsource/noto-sans": "^5.1.0",
     "@popperjs/core": "^2.5.3",
-    "@stencil-community/eslint-plugin": "^0.7.1",
-    "@stencil/angular-output-target": "^0.8.2",
-    "@stencil/react-output-target": "^0.5.3",
+    "@stencil-community/eslint-plugin": "0.x",
+    "@stencil/angular-output-target": "0.x",
+    "@stencil/react-output-target": "0.x",
+    "@stencil/vue-output-target": "0.x",
     "@stencil/sass": "^3.0.7",
-    "@stencil/vue-output-target": "^0.8.7",
-    "@types/jest": "^29.5.11",
     "@typescript-eslint/eslint-plugin": "^6.15.0",
     "@typescript-eslint/parser": "^6.15.0",
-    "jest": "^29.7.0",
```

Comment:
Why did you remove jest? 
There are still test specs for the different components?

Reply:
Had to move it to root package.json to make it work with the latest version. Specs are still running of course.